Satzung.md

```diff
@@ -0,0 +1,186 @@
+4. Wahl und Abwahl des Vorstands
+5. Änderungen der Satzung und Festlegung der Mitgliedsbeiträge
+6. Anträge des Vorstands und der Mitglieder
+7. Berufung/Beschwerde gegen die Ablehnung von Aufnahmeanträgen
+8. Ausschluss von Mitgliedern
+9. Vergabe und Aberkennung von Ehrenmitgliedschaften
+10. Auflösung des Vereins
+
+(2) Mitgliederversammlungen werden vom Vorstand in Textform, z.B. per E-Mail, unter Einhaltung einer Frist von einer Woche und Angabe der vorläufigen Tagesordnung einberufen. Einladungen gelten Mitgliedern als zugegangen, wenn sie an die letzte dem Verein in Textform bekanntgegebene E-Mail-Adresse gerichtet sind.
+
+(3) Der Vorstand muss jedes Jahr fristgemäß zu einer ordentlichen Mitgliederversammlung einladen, die an einem Tag im Zeitraum vom 25. bis zum 31. Januar stattfindet.
+
+(4) Der Vorstand kann jederzeit eine außerordentliche Mitgliederversammlung einberufen. Sie muss einberufen werden, wenn das Interesse des Vereins es erfordert oder wenn die Einberufung von einem Viertel aller Mitglieder in Textform, z.B. per E-Mail, unter Angabe der Gründe vom Vorstand verlangt wird. In letzterem Fall muss die außerordentliche Mitgliederversammlung spätestens vier Wochen nach Eingang des Antrags tagen.
+
+(5) Die Tagesordnung wird vom Vorstand festgesetzt. Jedes Mitglied des Vereins kann bis zum Beginn der Mitgliederversammlung beantragen, dass weitere Angelegenheiten, nicht jedoch der Ausschluss von Mitgliedern, Satzungsänderungen oder die Auflösung des Vereins, nachträglich auf die Tagesordnung gesetzt werden. Über die Annahme der Anträge beschließt die Mitgliederversammlung.
+
+
+## § 14 Beschlussfassung der Mitgliederversammlung
+
+(1) Die Mitgliederversammlung ist beschlussfähig, wenn mehr als die Hälfte der Mitglieder anwesend ist. Bei Beschlussunfähigkeit lädt der Vorstand umgehend zu einer zweiten Mitgliederversammlung mit gleicher Tagesordnung ein, die nach spätestens einem Monat tagt. Diese ist zu den in der Tagesordnung aufgeführten Punkten unabhängig von der Zahl der anwesenden Mitglieder beschlussfähig, wenn darauf mit der Einladung hingewiesen wird.
+
+(2) Die Leitung der Mitgliederversammlung wird vom Vorstand festgelegt. Bei Wahlen kann die Versammlungsleitung für die Dauer des Wahlgangs und der vorhergehenden Diskussion einem Wahlausschuss übergeben werden.
+
+(3) Jedes anwesende Mitglied des Vereins hat eine Stimme. Beschlüsse werden mit der einfachen Mehrheit der abgegebenen Stimmen gefasst, soweit die Satzung nichts anderes regelt. Bei Stimmengleichheit ist der Antrag abgelehnt.
+
+(4) Zur Änderung der Satzung ist eine Zweidrittelmehrheit der abgegebenen Stimmen erforderlich. Änderungen oder Ergänzungen der Satzung, die von der zuständigen Registerbehörde oder vom Finanzamt vorgeschrieben werden, werden vom Vorstand umgesetzt und bedürfen keiner Beschlussfassung durch die Mitgliederversammlung. Änderungen an der Satzung werden allen Mitgliedern des Vereins innerhalb einer Woche vom Vorstand mitgeteilt.
```

Comment:
Zu welchem Zeitpunkt sind Beschlüsse (insbesondere Satzungsänderungen) gültig? Eine sofortige Gültigkeit würde z.B. §14 (5) und (6) beeinflussen, da diese innerhalb einer Versammlung geändert und dann genutzt werden könnten.

Dementsprechend wäre es sinnvoll, eine explizite Gültigkeit (z.B. zum Ende der Versammlung) festzulegen.

Reply:
Da hab ich noch mal geschaut und das ist in § 71 Abs. 1 Satz 1 BGB geregelt: `Änderungen der Satzung bedürfen zu ihrer Wirksamkeit der Eintragung in das Vereinsregister.` 
Das heißt also, dass die Änderung nicht vor Ende der Sitzung inkrafttreten kann.

Reply:
Super, danke fürs Nachschlagen!